debugging/_index.md

```diff
@@ -0,0 +1,4 @@
+<!-- Template for the root page of a specification release -->
+---
+title: "Debugging Support for Other Languages Spec 1.0"
+date: 2019-07-18
```

Comment:
Remove the date entry.
Add a summary entry:

summary: "Jakarta Debugging Support for Other Languages provides a mechanism by which programs executed under the Java virtual machine but written in languages other than the Java programming language, can be debugged with references to the original source (for example, source file and line number references)."